.github/workflows/inductor-perf-test-nightly.yml

```diff
@@ -33,13 +63,40 @@ jobs:
           { config: "inductor_torchbench_perf", shard: 3, num_shards: 3, runner: "linux.gcp.a100.large" },
         ]}
 
-  linux-bionic-cuda11_8-py3_10-gcc7-inductor-test:
+  linux-bionic-cuda11_8-py3_10-gcc7-inductor-test-5days:
```

Comment:
I thought this means run every 5 days, but I realize now that it means run 5 times per day.  Please make sure that the job could finish within the hour from what I see.  Otherwise, I think the latter job has the chance of cancelling the previous running one I think (if they base on the same SHA commit)

Reply:
Oh, no, I got the string format wrong. Meant to run it 5 days a week (and 2 days a week for the other one). Let me fix it.